This test seems to require a D-Bus connection not available in the build
environment.

--- a/src/st/meson.build	2021-09-21 19:07:38.673049200 +0000
+++ b/src/st/meson.build	2021-09-26 18:23:38.461569826 +0000
@@ -191,18 +191,6 @@
   sources: st_enums[1]
 )
 
-test_theme = executable('test-theme',
-  sources: 'test-theme.c',
-  c_args: st_cflags,
-  dependencies: [mutter_dep, gtk_dep, libxml_dep],
-  build_rpath: mutter_typelibdir,
-  link_with: libst
-)
-
-test('CSS styling support', test_theme,
-  workdir: meson.current_source_dir()
-)
-
 libst_gir = gnome.generate_gir(libst,
   sources: st_gir_sources,
   nsversion: '1.0',
